lib/levelup.js

```diff
@@ -316,5 +283,20 @@ LevelUP.prototype.toString = function () {
   return 'LevelUP'
 }
 
+function getCallback (options, callback) {
+  return typeof options === 'function' ? options : callback
+}
+
+function getOptions (options) {
+  return typeof options === 'object' && options !== null ? options : {}
+}
```

Comment:
Moved this from `lib/util.js`. Maybe we can get rid of `lib/util.js` completely now.

Reply:
yeah, the promisify thing can go in npm :)